server/.env

```diff
@@ -38,10 +38,32 @@ GRAFANA_API_URL="https://grafana.liftedinit.tech"
 GRAFANA_USERNAME=username
 GRAFANA_PASSWORD=secretpassword
 PROMETHEUS_DATASOURCE_ID=datasourceid
+
 # JWT specific authentication. This should be changed.
 JWT_SECRET="DO NOT USE THIS VALUE. INSTEAD, CREATE A COMPLEX SECRET AND KEEP IT SAFE OUTSIDE OF THE SOURCE CODE."
 
 # Administrator. This cannot be changed once the application is running, and
 # is required.
 ADMIN_USERNAME=admin
 ADMIN_PASSWORD=admin
+
+# ====  Metrics SCHEDULER ====
+# Scheduler cron job running. If the scheduler hasn't finished work before the
+# next job, it will skip the schedule time.
+# If this and METRICS_SCHEDULER_SECONDS is missing, scheduler is disabled.
+# If both are set, this is an error.
+METRICS_SCHEDULER_CRON="*/60 * * * * *"
+
+# Maximum amount of metrics to fetch schedule.
+# Since this is blocking future jobs, this should be a reasonable amount
+# to fetch, as we also need to fetch the latest block to update the height
+# of the whole blockchain.
```

Comment:
Doesn't seem to apply to the following variable

Reply:
Will fix this comment!